app/views/site/about.html.erb

```diff
@@ -110,5 +128,9 @@ board_members = [
         <div class="card-deck the-board">
       <% end %>
     <% end %>
+
+    <% ((3 - board_members.size % 3) % 3).times do %>
+      <div class="card blank"></div>
+    <% end %>
```

Comment:
There has to be a better way to prevent the giant face of Tyler, but I'm tired.

Before:
![Screenshot 2020-07-08 10 05 06](https://user-images.githubusercontent.com/430986/86928415-82fdfc00-c102-11ea-9755-3e93fce8baab.png)

After:
![Screenshot 2020-07-08 10 05 38](https://user-images.githubusercontent.com/430986/86928471-9315db80-c102-11ea-9329-ce26ba639d6d.png)

Reply:
I think I speak for everyone when I say, I prefer the **Before**. 🤣

Reply:
Seriously though, you might like [`in_groups_of`](https://api.rubyonrails.org/classes/Array.html#method-i-in_groups_of) to handle this and get rid of the index checking in `if [2, 5, 8].include?(index)`.

Reply:
Ooooohhh neat, thanks! I'll try that.